storage/innobase/log/log0recv.cc

```diff
@@ -1866,11 +1866,6 @@ static byte *recv_parse_or_apply_log_rec_body(
               recv_sys->recovered_lsn > incremental_start_checkpoint_lsn) {
             full_scan_tables.insert(space_id);
           }
-          /* offline backup */
-          xb::info() << "Last flushed lsn: " << backup_redo_log_flushed_lsn
-                     << " load_index lsn " << recv_sys->recovered_lsn;
-
-          xb::warn(ER_IB_MSG_717);
```

Comment:
ER_IB_MSG_717 is used in another place?

Reply:
No. That was the only place. This leaves us with the message being declared and not used, but upstream has it declared in messages_to_error_log but does not use it anywhere.

Reply:
```
 🐬 marcelo  📂 /work/mysql/src  🔀 (mysql-8.0.32)  ▶ 
 ╰▶ $ grep -R ER_IB_MSG_717 .
./share/messages_to_error_log.txt:ER_IB_MSG_717

 🐬 marcelo  📂 /work/mysql/src  🔀 (mysql-8.0.32)  ▶ 
 ╰▶ $ 
```

Reply:
Ok, thanks, Even though we dont use it, we cannot remove it (due to shifting of error codes).
May be rename the 717 to ER_IB_MSG_717_NOT_USED?

Reply:
I am ok with that. Upstream currently have it as just ER_IB_MSG_717 . Will be minor, but might cause a merge conflict if they change it.

Reply:
Ok to leave it as is.